packages/storage/src/file.js

```diff
@@ -470,13 +472,11 @@ File.prototype.createReadStream = function(options) {
   // returned to the user.
   function makeRequest() {
     var reqOpts = {
-      uri: format('{downloadBaseUrl}/{bucketName}/{fileName}', {
-        downloadBaseUrl: STORAGE_DOWNLOAD_BASE_URL,
-        bucketName: self.bucket.name,
-        fileName: encodeURIComponent(self.name)
-      }),
+      uri: '',
```

Comment:
We have two classes we use throughout the Storage API.

"Service" (new Storage())
"ServiceObject" (new Bucket(), new File())

As part of their internally-defined methods, we get the `request` and `requestStream` functions. Since this is a File class, a `uri` value of empty string means the internal method won't add anything additional to the URI that's been pre-configured during the File object's instantiation.

Basically:

- Storage -> see "baseUrl": https://github.com/GoogleCloudPlatform/google-cloud-node/blob/ca0d96cb94f7ef4e176edf221f9ee154dcf8c850/packages/storage/src/index.js#L79
  - Bucket -> https://github.com/GoogleCloudPlatform/google-cloud-node/blob/ca0d96cb94f7ef4e176edf221f9ee154dcf8c850/packages/storage/src/bucket.js#L241
    - File -> https://github.com/GoogleCloudPlatform/google-cloud-node/blob/ca0d96cb94f7ef4e176edf221f9ee154dcf8c850/packages/storage/src/file.js#L267

The proper bucket and object name are inserted automatically.